Morcoso_Daisy/script.js

```diff
@@ -0,0 +1,84 @@
+document.addEventListener("DOMContentLoaded", function () {
```

Comment:
change the let into const (line 2-4 and line 23-24), since these codes are assigned once and never reassigned. Using const help to ensure that the values meant to stay constant.

Reply:
By the way, you can also fix the changes to your codes after your class. Thank you.